Aztec/Classes/Libxml2/DOM/Data/CommentNode.swift

```diff
@@ -35,5 +35,13 @@ extension Libxml2 {
         override func text() -> String {
             return String(.newline)
         }
+
+        override func deleteCharacters(inRange range: NSRange) {
```

Comment:
If you could merge the unit test I did on this PR https://github.com/wordpress-mobile/WordPress-Aztec-iOS/pull/370 to this branch it will be great.